### ISSUE TITLE
Preferences -> General -> Manage Plugins... not there

### ISSUE THREAD

**Issue by ekkis** [Free-GPGMail/Free-GPGMail#50]
first, thank you for maintaining this great tool.  I'm running on OSX Big Sur 11.4 and have just installed via brew.  everything went as expected and I can now see:
```bash
ls -alF /Users/ekkis/Library/Mail/Bundles
```
> drwxr-xr-x@ 3 ekkis  admin   96 Mar 17 14:54 Free-GPGMail_5.mailbundle/

but when I restart OSX Mail, I cannot perform step 2 of the installation.  the `Preferences -> General` doesn't offer a `Manage Plugins` button.  what am I missing?

**Appendix I**

interestingly, I decided to install the product from the https://gpgtools.org/ site and once installed and "activated", my `Mail/Bundles` directory looks the same... I expected an additional bundle.  the preferences dialogue does now show me the expected button.  I think it may be that the brew install is placing the bundle in the wrong place?

**Comment by bnavigator** [comment]
There is more than one bundle directory: User namespace and system namespace.

Here is a Big Sur (11.6) screenshot how it should look like:

![image](https://user-images.githubusercontent.com/4623504/167274212-ea374f8d-f7c9-4c45-b2d3-f0f52894b070.png)

**Comment by ekkis** [comment]
yes, the user space doesn't seem to be recognised by OSX Mail.  maybe there's some setting that needs turning on.  in fact, I moved the bundle brew installed to `/Library/Mail/Bundles` and removed the one that commercial product installed, and now I see a button.  I was able to grant the free version the permissions it needs

haven't tested to make sure it works but will update in a minute

**Comment by ekkis** [comment]
one other aspect of this is that when installing the commercial version it comes with GPG Chain, which is nice as otherwise there's no way to manage keys

**Comment by ekkis** [comment]
so I ran OSX Mail and I see the OpenGPG selector at the top right, but it doesn't allow me to encrypt messages.  the commercial version did indicate that encrypting wouldn't be allowed until I paid for it, but given that I've replaced the bundle, I expected I would be able to.  thoughts?

**Comment by bnavigator** [comment]
- Maybe you use a different user for mail than for the brew install.

- We specifically state that the bundle is only the liberated version of the Mail plugin. Everything else is only in the suite. That's why we recommend installing the suite but just leaving GPG Mail out. They didn't lock down the other open source components.

**Comment by ekkis** [comment]
same user, only user: `ekkis`

**Comment by bnavigator** [comment]
- you need a public key for the recipient.
- Does the preferences page say that you are in decrypt only mode? Are you sure that you enabled the Free-GPG Mail plugin and not the loader for GPG Mail?

**Comment by bnavigator** [comment]
https://help.mailbutler.io/help/manage-plug-ins-button-is-gone-from-apple-mail

**Comment by ekkis** [comment]
ah! I'm being stupid.  I have so many accounts.  it only works for the one on my keychain, as you astutely pointed out.  it seems to work now.  thanks a million!

**Comment by ekkis** [comment]
> That's why we recommend installing the suite but just leaving GPG Mail out

incidentally, I would just suggest stating that in the README.  it would have helped to know I should actually install the suite first

**Comment by bnavigator** [comment]
It *IS* in the README. Feel free to provide a pull request with a more concise wording.

Also, the [free-gpgmail cask](https://formulae.brew.sh/cask/free-gpgmail) already depends on [gpg-suite-no-mail](https://formulae.brew.sh/cask/gpg-suite-no-mail).
